app/bundles/LeadBundle/Translations/en_US/messages.ini

```diff
@@ -1016,6 +1016,10 @@ mautic.lead.field.notification.cannot_be_created_message="Custom field '%label%'
 mautic.lead.field.notification.cannot_be_created_header="Custom field cannot be created"
 mautic.lead.field.notification.custom_field_limit_hit_message="Custom field '%label%' cannot be created - you have reached the limit of custom fields allowed by your database."
 mautic.lead.field.notification.custom_field_limit_hit_header="Custom field cannot be created"
+mautic.segment.last_built="Last built"
+mautic.segment.last_built_duration_ms="Build time (milliseconds)"
+mautic.segment.not_built="Not yet built"
```

Comment:
Where is this being used?

Reply:
Indeed it needs removing, snuck in from a previous POC I think where it had an additional status before the build begin, but in review on our side it felt unnecessary to expose the fact that builds run on cron at specific times